src/app/features/event/event-map/event-map.component.ts

```diff
@@ -0,0 +1,58 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, inject, Input } from '@angular/core';
+import * as leaflet from 'leaflet';
+import { EventMapMarkerService } from './event-map-marker.service';
+
+@Component({
+  selector: 'app-event-map',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: `./event-map.component.html`,
+  styleUrls: ['./event-map.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EventMapComponent {
+  private map: leaflet.Map | leaflet.LayerGroup<any> | undefined;
+  private eventMapMarkerService = inject(EventMapMarkerService);
+  isCoordinates = true;
+  @Input() coordinates!: { altitude: number | null; latitude: number | null };
+
+  get getCoordinates() {
+    return this.coordinates;
+  }
+
+  private initMap() {
+    this.isCoordinates = true;
+
+    if (this.coordinates.altitude === null || this.coordinates.latitude === null) {
```

Comment:
A co się dzieje z mapką jak współrzędne są na `null`? Może dodamy jakiś inicjalny stan, wtedy można ominąć te przyrównywanie do nulli i takie typownie z opcjonalnymi wartościami?

Reply:
Gdy jest null nie wyświetla się mapka i pokazuje się komunikat 'Nie można załadować mapy!'. Inicjalny stan to właśnie dwa `null`e. Inicjalny stan leci ze store'a, a jeżeli serwer zwróci dwa `null`e to właśnie wyświetlamy błąd.

Reply:
Po rozmowie, doszliśmy do wniosku że inicjalny stan mapy powinien być nastawiony na siedzibę firmy

Reply:
Jednak zostajemy przy nullach